test/index.test.js

```diff
@@ -425,6 +410,13 @@ describe('Segment.io', function() {
           assert(!object._metadata);
         });
       });
+
+      it('should pick up messageId from AJS', function() {
```

Comment:
@f2prateek Is this along the lines of what you were looking for?

Reply:
Yep!